client/disputes/new-evidence/style.scss

```diff
@@ -48,26 +48,32 @@
 	&__stepper-section {
 		background: $white; // Same as .wcpay-accordion
 		border: $border-width solid $gray-200;
-		padding: 0 24px 128px 24px;
+		padding: 0 32px 64px 32px;
+		display: flex;
+		flex-direction: column;
 	}
 
 	// Limit the width of the stepper content and center it
 	&__stepper-content {
 		max-width: 500px;
 		margin-left: auto;
 		margin-right: auto;
+		display: flex;
+		flex-direction: column;
 	}
 
 	&__stepper-title {
 		@include heading-xl;
 		color: var( --Scales-Grays-gray-900, $gray-900 );
-		margin-top: 24px;
+		margin-top: 32px;
```

Comment:
when I look at the Figma file I see 24px, could you confirm if this is the right margin?

<img width="1822" alt="image" src="https://github.com/user-attachments/assets/f51ee4b1-1b67-40b2-967e-db521b72af2d" />